### PR TITLE
Add ViT data processing pipeline to hw_bench_script

### PR DESCRIPTION
**Pull request by JanuszL**
## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- Add ViT data processing pipeline to hw_bench_script

### Affected modules and functionalities:
- hw_bench_script

### Key points relevant for the review:
- NA

### Tests:
- NA

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Review by jantonguirao — APPROVED** [review]

File: tools/hw_decoder_bench.py [hunk 1]
```diff
@@ -140,6 +143,78 @@ def create_input_tensor(batch_size, file_list):
...
+            saturation=saturation)
+
+        # auto-augment
+        # `shape` controls the magnitude of the translation operations
+        img = auto_augment.auto_augment_image_net(img)
+    else:
+        img = fn.resize(img, size=image_shape[:-1])
+
+    # normalize
+    # https://github.com/NVIDIA/DALI/issues/4469
```
Comment (szalpal) [inline tools/hw_decoder_bench.py:204]: I believe it's not clear, why this issue is linked here (it is resolved and the resolution doesn't relate at the first glance to anything here). Could you put some more context in the comment here?

File: tools/hw_decoder_bench.py [hunk 2]
```diff
@@ -140,6 +143,78 @@ def create_input_tensor(batch_size, file_list):
...
+        img = fn.color_twist(
+            img,
+            brightness=brightness,
+            contrast=contrast,
+            hue=hue,
+            saturation=saturation)
+
+        # auto-augment
+        # `shape` controls the magnitude of the translation operations
+        img = auto_augment.auto_augment_image_net(img)
```
Comment (szalpal) [inline tools/hw_decoder_bench.py:199]: The comment mentions `shape` argument, however there's no such thing in the code itself. Does it relate to the `img` argument? Like 
```python
auto_augment.auto_augment_image_net(shape=img)
```
?

File: tools/hw_decoder_bench.py (hunk 1 above)
Comment (JanuszL) [reply tools/hw_decoder_bench.py:204]: It was taken as it is from https://github.com/awolant/JAX-Toolbox/blob/f875b96d7d66916c6e4125f8315ddc197debf677/rosetta/rosetta/projects/vit/dali_update_test.py#L112.

File: tools/hw_decoder_bench.py (hunk 2 above)
Comment (JanuszL) [reply tools/hw_decoder_bench.py:199]: It was taken as it is from https://github.com/awolant/JAX-Toolbox/blob/f875b96d7d66916c6e4125f8315ddc197debf677/rosetta/rosetta/projects/vit/dali_update_test.py#L112.

File: tools/hw_decoder_bench.py (hunk 1 above)
Comment (JanuszL) [reply tools/hw_decoder_bench.py:204]: Anyway - removed comments.

File: tools/hw_decoder_bench.py (hunk 2 above)
Comment (JanuszL) [reply tools/hw_decoder_bench.py:199]: Anyway - removed comments.

**Review by szalpal — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Pull request merged**